atlassian-terraform/terraform.tfvars

```diff
@@ -0,0 +1,93 @@
+# Sumo Logic - Atlassian Terraform
```

Comment:
I am a bit worried about this file - there are so many variables here, easy to miss one. For all MANDATORY variables or things people need , can we use a syntax like <> and mention that people need to provide values for those places, INCLUDING the angle brackets

For example: 
environment ="<deployment>"  #  please provide the deployment for your Sumo org

Reply:
Added.